src/Frontend/Core/Engine/FormExtension.php

```diff
@@ -0,0 +1,380 @@
+<?php
+namespace Frontend\Core\Engine;
```

Comment:
I don't think this class is correct PSR2 yet. (some coding style issues + multiple classes in one file that have not the same name as the file)